Makefile

```diff
@@ -97,6 +97,25 @@ endef
 $(foreach t, $(sort $(wildcard ta/*/user_ta.mk)), $(eval $(call build-user-ta,$(t))))
 endif
 
+# Platform/arch config is supposed to assign the targets
+sp-targets ?= invalid
+default-sp-target ?= $(firstword $(sp-targets))
```

Comment:
these 3 lines should go with the conditioned instructions below.

Reply:
Done